pillar/environment-prod-public.sls

```diff
@@ -13,3 +13,7 @@ elife_xpub:
     s3:
         bucket: prod-elife-xpub
     deployment_target: prod
+
+journal:
+    feature_xpub: false
+    submit_url: https://xpub.elifesciences.org/login
```

Comment:
This needs to remain EJP, https://github.com/elifesciences/journal/pull/1073 will use this value for the 'Submit my research' link if `feature_xpub` is `false`.